spec/frozen_record_spec.rb

```diff
@@ -55,11 +55,16 @@
 
   describe '#load_records' do
 
-    it 'processes erb' do
+    it 'processes erb by default' do
       country = Country.first
       expect(country.capital).to be == 'Ottawa'
     end
 
+    it 'loads records with a custom backend' do
```

Comment:
Just this one test so far, let me know what would you think is a good approach for testing.